__init__.py

```diff
@@ -22,4 +22,4 @@
   from . import nn  # noqa
 
 else:
-  nn = _LazyLoader("nn", globals())
```

Comment:
First of all, whatever you try to fix/change here should be a separate PR.

Then further, this is obviously wrong. But let's discuss that in the separate PR.

Reply:
Hmm I think I see why this might be wrong, but then I am not really sure how else to fix it.